queue/queue.go

```diff
@@ -27,6 +27,9 @@ type Store struct {
 func (db *Store) Next(ctx context.Context, resp mdm.Response) (*Command, error) {
 	dc, err := db.DeviceCommand(resp.UDID)
 	if err != nil {
+		if isNotFound(err) {
```

Comment:
Confused here. Are you proposing an `else if` here? Or two separate if statements the other of which catches the other error cases (the `Wrapf()` return)?